lib/routes/revision.js

```diff
@@ -0,0 +1,56 @@
+module.exports = function(crowi, app) {
+  'use strict';
+
+  var debug = require('debug')('crowi:routes:revision')
+    , Revision = crowi.model('Revision')
+    , ApiResponse = require('../util/apiResponse')
+    , actions = {}
+  ;
+  actions.api = {};
+
+  /**
+   * @api {get} /revision.get Get revision
```

Comment:
as a definition of routes, this should be `revisions.get` (also `revisions.list`)